lightning-invoice/src/utils.rs

```diff
@@ -80,42 +77,34 @@ pub fn create_phantom_invoice<Signer: Sign, K: Deref>(
 ///
 /// `description_hash` is a SHA-256 hash of the description text
 ///
-/// `payment_hash` and `payment_secret` can come from [`ChannelManager::create_inbound_payment`] or
+/// `payment_hash` can come from [`ChannelManager::create_inbound_payment`] or
 /// [`ChannelManager::create_inbound_payment_for_hash`]. These values can be retrieved from any
-/// participating node. Alternatively, [`inbound_payment::create`] or
-/// [`inbound_payment::create_from_hash`] may be used to retrieve these values without a
-/// `ChannelManager`.
+/// participating node. If `None` is provided for `payment_hash`, then one will be created.
```

Comment:
Can you add a note in the docs about what `invoice_expiry_delta_secs` is. One sentence should suffice, it is pretty clear, but best to always write it down imo.